Cargo.toml

```diff
@@ -11,19 +11,21 @@ homepage = "https://zama.ai/"
 keywords = ["fft"]
 
```

Comment:
Can we bump the version to 0.2.0 so that we don't have license issues for the existing TFHE-rs, also it's such a big code change I think it makes sense

Reply:
yeah, bumping to 0.2.0 makes sense. what kind of license issues are you referring to?

Reply:
Sorry, meant to avoid changing the dependency tree so that TFHE-rs 0.1.x keeps the same deps/licenses in its dep tree

Reply:
done